doc/source/getting_started/installation.rst

```diff
@@ -1,92 +1,104 @@
-.. _installation:
+.. _ref_installation:
 
 ============
 Installation
 ============
 
-The ``ansys-fluent-core`` package supports Python 3.9 through
-Python 3.12 on Windows and Linux.
 
-.. note::
-   This page provides information for quickly installing and launching
-   the ``ansys-fluent-core`` package. Additional PyFluent packages, which
-   are described later on this page, can also be installed. For
-   information on installing all PyFluent packages in a virtual environment,
-   see :ref:`faqs_install` in :ref:`faqs`.
+PyFluent Installation
+---------------------
 
-Install the package
--------------------
-Install the latest ``ansys-fluent-core`` package from
-`PyPi <https://pypi.org/project/ansys-fluent-core/>`_ with this code:
+PyFluent supports Python 3.9 through Python 3.12 on Windows, Mac OS and Linux.
 
-.. code::
+PyFluent can be installed, along with all its optional dependencies, using:
 
+.. code:: console
    pip install ansys-fluent-core
 
-If you plan on doing local *development* of PyFluent with Git, install the
-latest ``ansys-fluent-core`` package with this code:
+
+Development Installation
+------------------------
+The PyFluent source repository is available on GitHub. You can clone the repository and set up for local
+development with the following commands:
 
 .. code:: console
 
    git clone https://github.com/ansys/pyfluent.git
    cd pyfluent
    pip install pip -U
-   pip install -e .
-   python codegen/allapigen.py  # Generates the API files
+   pip install -e
```

Comment:
Just checked, this will require the trailing dot.

pip install [options] [-e] <local project path> ...

Reply:
You're right, if trying without the `.` I get the error: "-e option requires 1 argument"

So this should be:
```suggestion
   pip install -e .
```